### DISCUSSION TITLE
Karma money

### DISCUSSION THREAD

**Discussion by TheBojda** [StableCredit/stable-credit#4] [General]
Hello Guys!

I have a similar project: Karma Money. Your solution is more professional, but maybe my pet project can give some inspiration.

The project: https://github.com/TheBojda/karma_money

Medium article about it: https://medium.com/better-programming/karma-an-erc20-compatible-alternative-money-on-the-ethereum-blockchain-cee660b821ce

**Comment by teakmodz** [comment]
Does this make money in the Erc-20 realm
